packages/react/carbon.yml

```diff
@@ -0,0 +1,699 @@
+      thumbnailPath: '' 
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/components-unorderedlist
+      # TODO: @Aubrey - double check tags
+      tags:
+        - content-block
+        - data-display 
+    grid:
+      status: stable
+      type: component
+      framework: react
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/elements-grid
+    fluid-form:
+      name: Fluid form
+      framework: react
+      status: experimental
+      type: component
+      platform: web
+      # TODO: add thumbnail
+      thumbnailPath: '' 
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/experimental-fluidform
+      # TODO: @Aubrey - double check tags
+      tags:
+        - content-block
+    unstable-heading:
+      name: Unstable heading
+      framework: react
+      status: experimental
+      type: component
+      platform: web
+      # TODO: add thumbnail
+      thumbnailPath: '' 
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/experimental-unstable-heading
+      # TODO: @Aubrey - double check tags
+      tags:
+        - content-element
+    unstable-menu:
+      name: Unstable menu
+      framework: react
+      status: experimental
+      type: component
+      platform: web
+      # TODO: add thumbnail
+      thumbnailPath: '' 
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/experimental-unstable-menu
+      # TODO: @Aubrey - double check tags
+      tags:
+        - content-block
+        - data-display
+    unstable-pagination:
+      name: Unstable pagination
+      framework: react
+      status: experimental
+      type: component
+      platform: web
+      # TODO: add thumbnail
+      thumbnailPath: '' 
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/experimental-unstable-pagination
+      # TODO: @Aubrey - double check tags
+      tags:
+        - data-display
+        - contextual-navigation
+    unstable-text:
+      name: Unstable text
+      framework: react
+      status: experimental
+      type: component
+      platform: web
+      # TODO: add thumbnail
+      thumbnailPath: '' 
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://react.carbondesignsystem.com/?path=/docs/experimental-unstable-text
+      # TODO: @Aubrey - double check tags
+      tags:
+        - content-element
```

Comment:
Keep content-element. Add input-control (some examples in the docs allow the user to toggle between alignments, I think)